bindings/node.js/kzg.cxx

```diff
@@ -32,41 +33,67 @@ Napi::Value throw_invalid_argument_type(const Napi::Env env, std::string name, s
   return env.Null();
 }
 
-Napi::TypedArrayOf<uint8_t> napi_typed_array_from_bytes(uint8_t* array, size_t length, Napi::Env env) {
-  // Create std::vector<uint8_t> out of array.
-  // We allocate it on the heap to allow wrapping it up into ArrayBuffer.
-  std::unique_ptr<std::vector<uint8_t>> vector =
-      std::make_unique<std::vector<uint8_t>>(length, 0);
-
-  for (size_t i = 0; i < length; ++i) {
-    (*vector)[i] = array[i];
+/**
+ * Checks for:
+ * - arg is Uint8Array or Buffer (inherits from Uint8Array)
+ * - underlying ArrayBuffer length is correct
+ * 
+ * Internal function for argument validation. Prefer to use
+ * the helpers below that already have the reinterpreted casts:
+ * - get_blob
+ * - get_commitment
+ * - get_proof
+ * - get_bytes_32
+ * 
+ * Built to pass in a raw Napi::Value so it can be used like
+ * `get_bytes(env, info[0])` or can also be used to pull props from
+ * arrays like `get_bytes(env, passed_napi_array[2])`
+ * 
+ * Designed to raise the correct javascript exception and return a
+ * valid pointer to the calling context to avoid native stack-frame
+ * unwinds.  Calling context can check for `nullptr` to see if an
+ * exception was raised or a valid pointer was returned from V8
+ * 
+ * @param[in] env - napi_env passed from calling context
+ * @param[in] val - napi_value
+ * @param[in] length - size_t to validate ArrayBuffer against
+ * @param[in] name - name of prop being validated for error reporting
```

Comment:
Nit, these docstrings don't match the style of the others. Could we update?

Reply:
Sure! Updated.

Reply:
Sure!  Updated.